### ISSUE TITLE
Programação Funcional é uma Seita

### ISSUE THREAD

**Issue by lincolixavier** [ospilhacheia/tretadev#16]
Boatos que fazem pentagramas invertidos com sangue, pra sacrifícios pra Deusa Lambda

**Comment by wilcorrea** [comment]
![image](https://user-images.githubusercontent.com/803733/118053247-8da8a380-b35a-11eb-8cf9-53173605d304.png)

alimentando os tubarões

**Comment by wilcorrea** [comment]
as melhores partes foram apagadas 😢

**Comment by lincolixavier** [comment]
Droga, pelo menos eu li antes :P

**Comment by LeonardoFurtado** [comment]
> Droga, pelo menos eu li antes :P

eu perdi, cheguei tarde pra treta

**Comment by wilcorrea** [comment]
@LeonardoFurtado eu juro que li alguém dizer algo como: "ninguém fala que programação funcional é melhor que qualquer outra coisa, mas programação funcional é melhor que qualquer outra coisa que existe na face da terra e o react é o exemplo vivo disso"

**Comment by LeonardoFurtado** [comment]
> @LeonardoFurtado eu juro que li alguém dizer algo como: "ninguém fala que programação funcional é melhor que qualquer outra coisa, mas programação funcional é melhor que qualquer outra coisa que existe na face da terra e o react é o exemplo vivo disso"

Linus Torvalds não concorda com isso

**Comment by codeferreira** [comment]
Mano eu curto programação funcional e adoro Elixir! Mas a galera ta passando do limite pensando ser bala de prata

**Comment by jmurowaniecki** [comment]
**Programação é** Seita.

**Comment by rodrigogs** [comment]
O problema é quando o dev desiste do projeto e pede demissão porque não vai ser em linguagem funcional 🤣

**Comment by terremoth** [comment]
Sim, essa seita é liderada pelo Map, Filter e Reduce, cujos filhos são respectivamente Lambda, Mônadas e Functores
